### DISCUSSION TITLE
Problem med udregning - Syntax error

### DISCUSSION THREAD

**Discussion by kaolinx** [Eduap-com/WordMat#204] [Support]
Hej, er stødt på et problem med når man skal udregne bestemte ligninger og ved ikke lige hvad jeg skal gøre længere. Har prøvet at genstarte, geninstallere og prøver at skifte mellem de forskellige indstilling.
![Fejl](https://github.com/Eduap-com/WordMat/assets/72488701/881f90cf-beda-4838-a3ee-aca80fad55d9)

**Comment by GGHEPE** [comment]
Jeg her en elev med et lignende problem:

Syntax error når man anvender gangetegn.
På windows 11, Office 365, opleves det at når man anvender gangetegn, så laver wordmat en syntax fejl.
Det er afprøvet med version 1.27, 1,28, 1,29 og fejlen er konsekvent.
Selv hvis man anvender gangetegnene under ligninger, skriver wordmat, at den ikke kan beregne udtrykket.
Ved en simpel udregning af 10*10 svarer wordmat: times 10 is not an infix operator 10\times10.
